src/pages/UpSertUser/UpdatePage/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useHistory, useParams } from "react-router-dom";
+
+import { CircularProgress } from "@material-ui/core";
+import { IUser } from "src/components/UserCard";
+
+import UpSertPage from "../UpSertPage";
+import useGetSpecificUser from "../hooks/useGetSpecificUser";
+
+import useUpdateUser from "src/pages/Users/hooks/useUpdateUser";
+
+const UpdatePage = () => {
+    const { id } = useParams<{ id: string }>();
+    const { data, isLoading } = useGetSpecificUser(id);
+
+    const history = useHistory();
+
+    const backToListPage = () => {
+        history.push("/user");
+    };
+
+    const { mutate } = useUpdateUser(backToListPage);
+
+    const sendDataToServer = (user: IUser) => {
+        mutate({
+            name: user.name || "",
```

Comment:
Please validate instead of sending blank to db

Reply:
![image](https://user-images.githubusercontent.com/58205909/131253220-139ae896-da4e-416d-9408-30f4e3b8b897.png)